src/main/java/io/icker/factions/api/persistents/Home.java

```diff
@@ -25,6 +29,9 @@ public class Home {
 
     private UUID factionID;
 
+    @Field("Home Cooldown")
+    public HashMap<ServerPlayerEntity, Long> HomeCooldown = new HashMap<>();
```

Comment:
Our data serialization cannot serialize or serialize `HashMap`s or `ServerPlayerEntity`s. Regardless, I think it makes sense to record the last home cooldown time in the user instead.